doozerlib/cli/images_upstreampulls.py

```diff
@@ -0,0 +1,62 @@
+# stdlib
+import os
+import yaml
+
+# external
+from github import Github
+
+# doozerlib
+from doozerlib import constants
+from doozerlib.cli import cli, pass_runtime
+
+github_token = os.getenv(constants.GIT_TOKEN)
+github_client = Github(github_token)
+repo = github_client.get_repo("openshift/ocp-build-data")
```

Comment:
`openshift/ocp-build-data` shouldn't be hardcoded here. How could I run the doozer with my own fork of ocp-build-data?

Global variables should be avoided as long as you can.